pkg/stub/status.go

```diff
@@ -65,16 +66,18 @@ func (h *Handler) syncOperatorStatus() {
 		},
 	}
 
+	versionMap, err := getVersionMap()
+	if err != nil {
+		logrus.Errorf("syncOperatorStatus: getVersionMap: %v", err)
+
```

Comment:
Newline before return is the convention we follow?

Reply:
Blank line before `return` is a convention I learned back in my C days, but we have not been observing it consistently.  The blank line makes function exit points stand out, but omitting it would make the code more compact.  I'm open to changing it if we decide to go that way.

Reply:
I went ahead and deleted the blank lines.